pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/AbstractJavaRule.java

```diff
@@ -96,8 +96,12 @@ public static boolean importsPackage(ASTCompilationUnit node, String packageName
         return false;
     }
 
+    /**
+     * @deprecated Used in only one deprecated rule, which is removed by pending PR #1772
```

Comment:
#1772 is now merged… shall this method be removed altogether and deprecated in master?

Reply:
In fact that comment is wrong, the PR removing that rule is #2022 (rules are the old  [Std|Modified]CyclomaticComplexity). But I deprecated the method in #2031